src/price_data.py

```diff
@@ -26,8 +26,10 @@
 import config
 import misc
 from transaction import *
+from core import kraken_pair_map
 
 log = logging.getLogger(__name__)
+logging.getLogger("urllib3").propagate = False
```

Comment:
You can decide if you want to keep those urllib debug lines in the log. I found them quite distracting, so this is the code to disable these.

Reply:
Absolutely. :+1: I seem to have overlooked that file altogether…